test/test-layout.js

```diff
@@ -30,3 +38,13 @@ describe('dummy layout engine', function() {
     done();
   });
 });
+
+
+describe('spring layout engine', function() {
+  it('should take a graph and not blow up', function(done) {
+    var testGraph = generateGraph();
+    SpringLayoutEngine(graph);
+    testGraph.should.be.an.instanceOf(Graph);
+    done();
+  });
+});
```

Comment:
It might be good to add a test that ensures that nodes are not positioned identically. You mentioned that this might be a problem.